Rubberduck.RegexAssistant/Atom.cs

```diff
@@ -0,0 +1,284 @@
+﻿using Rubberduck.RegexAssistant.i18n;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Rubberduck.RegexAssistant
+{
+    public interface Atom : IDescribable
```

Comment:
Shoops. That one started out as an abstract implementation of IRegularExpression instead of SingleAtomExpression that's why there's no I there